sdk/python/kfp/dsl/__init__.py

```diff
@@ -25,3 +25,6 @@
 from ._ops_group import OpsGroup, ExitHandler, Condition, ParallelFor
 from ._component import python_component, graph_component, component
 from ._artifact_location import ArtifactLocation
+
+execution_id_placeholder = '{{workflow.uid}}-{{pod.name}}'
```

Comment:
How about making it capitalized, EXECUTION_ID_PLACEHOLDER, because it's constant?

Reply:
This is probably a good idea.
I've uppercased both constants.